src/chapters/exceptions/exceptions-control-flow.rst

```diff
@@ -1,71 +1,137 @@
-Exceptions-Control-Flow
-=======================
+Exceptions as Control Flow
+==========================
+Runtime errors occur as the program runs. Runtime errors are also know as exceptions. Exceptions
+are caused by referencing undeclared variables and invalid or unexpected data.
 
-In some instances, we receive input from a user and we cannot always ensure they will type in something that will run with our code.
 
-JavaScript gives us some control flow tools to handle exceptions by catching thrown exceptions.
+Control Flow
+------------
+.. index:: ! control flow
 
-We may be anticipating an exception, but still want our program to continue running and not experience a runtime error. We can tell JavaScript to **try** to run a block of code and if an exception is thrown, to **catch** the exception and run a different block of code.
+**Control flow** of a program is the order in which the statements are executed. Normal control flow
+runs from top to bottom of a file. An exception breaks the normal flow and stops the program. A stopped
+program can no longer interact with the user. Luckily JavaScript provides a way to anticipate and
+handle exceptions.
+
+
+Catching an Exception
+---------------------
+.. index:: ! try catch
+
+JavaScript provides ``try`` and ``catch`` statements that allow us to keep our programs running even if there
+is an exception. We can tell JavaScript to *try* to run a block of code and if an exception is thrown, to *catch* the
+exception and run a different block of code. Anticipating and catching the exception makes the exception now part of
+the control flow.
+
+.. note:: Catching an exception is also known as *handling* an exception.
 
 .. admonition:: Example
 
-   .. sourcecode:: js
+   In this example there is an array of animals. The user is asked to enter the index for the animal they want to see.
+   If the user enters an index that does NOT contain an animal, the code will try throw an ``TypeError`` when ``name``
```

Comment:
Grammar/phrasing: Change "...the code will try throw an ``TypeError``..." to "...the code will throw a ``TypeError``..."